modules/firehose/sync.go

```diff
@@ -3,15 +3,29 @@ package firehose
 import (
 	"context"
 	"encoding/json"
+	"time"
 
 	"github.com/odpf/entropy/core/module"
 	"github.com/odpf/entropy/core/resource"
+	"github.com/odpf/entropy/modules/firehose/kafka"
 	"github.com/odpf/entropy/modules/kubernetes"
 	"github.com/odpf/entropy/pkg/errors"
 	"github.com/odpf/entropy/pkg/helm"
+	"github.com/odpf/entropy/pkg/kube"
+	"github.com/odpf/entropy/pkg/worker"
 )
 
-func (m *firehoseModule) Sync(_ context.Context, spec module.Spec) (*resource.State, error) {
+const (
+	networkErrorRetrySeconds = 5
```

Comment:
```suggestion
	networkErrorRetryDuration = 5 * time.Second
```
Somehow unable to find this suggestion on Github UI, so adding it again.